sky/shell/platform/mojo/BUILD.gn

```diff
@@ -44,4 +49,11 @@ mojo_native_application("mojo") {
     "//third_party/icu",
     "//url",
   ]
+
+  if (is_android) {
+    ldflags = [
+      "-lGLESv2",
```

Comment:
Yeah, we don't use that function, but it's still made that it's symbols are missing.